docs/projects/project-setup.md

```diff
@@ -215,4 +215,22 @@ train_function = project.set_function(
 )
 train_function.with_limits(gpus=gpus, cpu=cpu, mem=mem)
 train_function.save()
-```
\ No newline at end of file
+```
+
+### Loading project from template
```

Comment:
### Loading a project from a template
You can load a project from a template only if one of the following conditions is met:
- Set the `allow_cross_project` flag = True and change the name of the project.
- Change the name in the yaml file or delete it.
- Change the context dir .